source/unified-test-format/tests/Makefile

```diff
@@ -1,8 +1,8 @@
-SCHEMA=../schema-1.15.json
+SCHEMA=../schema-1.18.json
 
 .PHONY: all invalid valid-fail valid-pass versioned-api load-balancers gridfs transactions crud collection-management sessions command-logging-and-monitoring client-side-operations-timeout HAS_AJV
 
-all: invalid valid-fail valid-pass versioned-api load-balancers gridfs transactions change-streams crud collection-management sessions command-logging-and-monitoring client-side-operations-timeout
+all: invalid valid-fail valid-pass versioned-api load-balancers gridfs transactions change-streams crud collection-management sessions command-logging-and-monitoring client-side-operations-timeout client-side-encryption
```

Comment:
These aren't the first unified tests for CSFLE. This makes me wonder how the existing tests were converted.